Sources/GRPC/HTTP1ToGRPCServerCodec.swift

```diff
@@ -128,13 +128,19 @@ extension HTTP1ToGRPCServerCodec: ChannelInboundHandler {
   public typealias InboundOut = _RawGRPCServerRequestPart
 
   public func channelRead(context: ChannelHandlerContext, data: NIOAny) {
+    let unwrappedData = self.unwrapInboundIn(data)
     if case .ignore = self.inboundState {
+        if case let .body(body) = unwrappedData, body.readableBytes == 0 {
+            return // Ignoring a read of zero bytes is always fine.
+        } else if case .end = unwrappedData {
+            return // Ignoring an end stream is not an event worth reporting.
+        }
       self.logger.notice("ignoring read data", metadata: ["data": "\(data)"])
       return
```

Comment:
Can we `switch` on the `unwrappedData` here instead? We can drop our `return` count to 1 and neaten it up a bit. 

Something like:

```swift
switch unwrappedData {
  case let .body(body) where body.readableBytes == 0:
    ()
  case .end:
    ()
  default:
    self.logger.notice(...)
  }
  return
}
```

Reply:
Also we may as well log the `unwrappedData` instead of the `data` now.

Reply:
Also you'll need to run `./scripts/format.sh`.